packages/twenty-front/src/modules/object-record/object-filter-dropdown/utils/configurableViewFilterOperands.ts

```diff
@@ -0,0 +1,14 @@
+import { ViewFilterOperand } from '@/views/types/ViewFilterOperand';
+
+export const configurableViewFilterOperands = new Set<ViewFilterOperand>([
```

Comment:
Is using a Set necessary here ? It can be harder to manipulate than a simple array, for a unicity constraint that is of marginal usage since we already control the constant definition here ?